package.json

```diff
@@ -9,7 +9,7 @@
     "new-error": "plop error",
     "new-test": "plop test",
     "clean": "yarn lerna clean -y && yarn lerna bootstrap && yarn lerna exec 'rm -rf ./dist'",
-    "build": "yarn prepublish",
+    "build": "yarn prepublishOnly",
```

Comment:
Should all of these `yarn` references be changed to `pnpm`?

Reply:
Yup, updated in https://github.com/vercel/next.js/pull/37259/commits/56c938ce052d810d586f4ea494793362f42a2225